bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx109-en.html

```diff
@@ -0,0 +1,64 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% extends "firefox/whatsnew/base.html" %}
+
+{% block page_title %}{{ ftl('whatsnew-page-title') }}{% endblock %}
+{% block page_desc %}{{ ftl('whatsnew-page-description') }}{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('firefox_whatsnew') }}
+  {{ css_bundle('firefox_whatsnew_109_en') }}
+{% endblock %}
+
+{% block experiments %}
+  {% if switch('experiment-firefox-whatsnew-109-en') %}
+    {{ js_bundle('firefox_whatsnew_109_en_experiment') }}
+  {% endif %}
+{% endblock %}
```

Comment:
Drive by comment: we should make sure this block/bundle is in both the control and the variation templates, as it's needed to fire the `data-ex-name` and `data-ex-variant` events on page load.